cpu/nrf52/Makefile.include

```diff
@@ -1,5 +1,9 @@
 export CPU_ARCH = cortex-m4f
 export CPU_FAM  = nrf52
 
+# Export internal ROM alignment and image header size for bootloader support
+export IMAGE_ALIGN = 8
+export IMAGE_HDR_SIZE = 512
```

Comment:
OK, I agree, I didn't find the right place for such variable but we can put it there if that's the most convenient.